styles/popper.css

```diff
@@ -197,6 +198,7 @@
 
 .popper .citation-editor,
 .popper .table-footnote-editor {
+  position: relative;
```

Comment:
please move the relative position just to `citation-editor` element, as that will affect `table-footnote-editor`